docs/resources/instance_server.md

```diff
@@ -140,6 +142,10 @@ Updates to this field will recreate a new resource.
 - `image` - (Required) The UUID or the label of the base image used by the server. You can use [this endpoint](https://api-marketplace.scaleway.com/images?page=1&per_page=100)
 to find either the right `label` or the right local image `ID` for a given `type`.
 
+You can check the available labels with our [CLI](https://www.scaleway.com/en/docs/compute/instances/api-cli/creating-managing-instances-with-cliv2/). ```scw marketplace image list```
+
+To retrieve more information by label please use: ```scw marketplace image get label=<LABEL>```
+
 [//]: # (TODO: Improve me)
```

Comment:
Can we delete this TODO? Or at least restructure it and open a ticket if some content is missing?

Reply:
TODO removed